torch/_inductor/kernel/flex_attention.py

```diff
@@ -156,7 +184,7 @@ def build_subgraph_buffer(
     name="flex_attention",
     grid=flex_attention_grid,
     source=r"""
-{{def_kernel("Q", "K", "V", "LSE")}}
+{{def_kernel("Q", "K", "V", "LSE", "SM_KV_NUM_BLKS", "SM_KV_IDX")}}
```

Comment:
SM? Assuming it's not streaming-multiprocessor, it's kind of a confusing name to me.

Reply:
Updated to ```SPARSE_KV_NUM_BLKS```